app/src/main/java/com/android/bookswap/ui/map/FilterMap.kt

```diff
@@ -89,18 +90,22 @@ fun FilterMapScreen(navigationActions: NavigationActions, selectedFilters: Mutab
         LazyColumn(contentPadding = paddingValues, modifier = Modifier.fillMaxSize()) {
           item {
             ButtonBlock(
-                BookGenres.values().map {
-                  it.toString().lowercase().replaceFirstChar { c -> c.uppercase() }
-                },
-                selectedFiltersTemp)
+                BookGenres.values().map { it.Genre }, selectedFiltersGenres.map { it.Genre }) {
```

Comment:
I seem to miss some alignment modifiers, maybe some padding. Through this I cannot sight what the final screen may look like, but it seems like some standardised padding and some central alignment may benefit. That being said, and as you may with great ease realise, it may look quite alright as of now. If that is the case, please ignore this message.

Reply:
Fixed  in the FlowRow